src/test/java/JavaJDAExample.java

```diff
@@ -214,6 +214,20 @@ public void onSlashCommandInteraction(@NotNull SlashCommandInteractionEvent even
                     }
                 });
                 break;
+            }
+            case "custom-json-request": {
+                final Link link = this.client.getLink(event.getGuild().getIdLong());
+                link.getNode().customJsonRequest(LoadResult.Serializer.INSTANCE,
+                        (builder) -> builder.path("/v4/loadtracks?identifier=ytsearch%3Anever%20gonna%20give%20you%20up").get()
+                ).subscribe((loadResult -> {
+                    if (loadResult instanceof LoadResult.SearchResult searchResult) {
+                        event.reply("Loaded " + searchResult.getData().getTracks().size() + " tracks!").queue();
+                    } else if (loadResult instanceof LoadResult.LoadFailed loadFailed) {
+                        event.reply("Failed to load tracks! " + loadFailed.getData().getMessage()).queue();
+                    }
+                }));
+                break;
```

Comment:
This clutters the code a lot, maybe you can add lavasearch as a test dep and show an example using that in a separate class